src/EventHub/EventHub.Autorest/custom/New-AzEventHubNamespaceV2.ps1

```diff
@@ -0,0 +1,216 @@
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        ${DefaultProfile},
+
+        [Parameter(HelpMessage = "Run the command as a job")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Run the command as a job
+        ${AsJob},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Wait for .NET debugger to attach
+        ${Break},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline
+        ${HttpPipelineAppend},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+        ${HttpPipelinePrepend},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Run the command asynchronously
+        ${NoWait},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use
+        ${Proxy},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call
+        ${ProxyCredential},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy
+        ${ProxyUseDefaultCredentials}
+    )
+    process{
+		try{
+            $null = $PSBoundParameters.Remove('WhatIf')
+            $null = $PSBoundParameters.Remove('Confirm')
+            if($PSBoundParameters.ContainsKey('SkuName')){
+                $PSBoundParameters.Add('SkuTier', $SkuName)
+            }
+
+            if($PSBoundParameters.ContainsKey('KeyVaultProperty')){
+                $PSBoundParameters.Add('KeySource', 'Microsoft.KeyVault')
+            }
+            
+            if($PSBoundParameters.ContainsKey('UserAssignedIdentityId')){
+                $identityHashTable = @{}
+
+			    foreach ($resourceID in $UserAssignedIdentityId){
```

Comment:
indentation problem

Reply:
![image](https://user-images.githubusercontent.com/91929893/203923692-80fde220-79d2-4016-baad-e4d0d22e634c.png)
In my local Visual Studio, I find these are well indented.

Reply:
I see that IDE's and GitHub conflict during identation. If I indent directly on GitHub I see that my code on local becomes unindented after pull.

Reply:
![image](https://user-images.githubusercontent.com/91929893/203925556-682fdd91-b172-4e0e-b925-307f2d309719.png)

Reply:
Hi @damodaravadhani , please use a space to indent your code. The tab displays with different wide in different IDE.